app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(name=i["name"], age=i["age"]) for i in people]
+    Person.people = {exemplar.name: exemplar for exemplar in person_list}
```

Comment:
make more informative name for variable "i"
u can add person to the people dict inside init method - no need to make +1 loop for this